app/models/cita_sync/persist.rb

```diff
@@ -172,8 +173,8 @@ def save_blocks_with_infos
       def realtime_sync
         loop do
           save_blocks_with_infos
+          sleep(ENV["LOOP_INTERVAL"].to_f)
```

Comment:
In case we forget setting `LOOP_INTERVAL`, it would fallback to `sleep(0)`. Is that expected?
Or perhaps fallback to a default value > 0?

Reply:
Yeah, overwrite it in `.env.local` anything you want.